docs/data/toolpad/core/components/sign-in-page/OAuthSignInPage.tsx.preview

```diff
@@ -0,0 +1,4 @@
+<SignInPage
```

Comment:
Would be useful if we could see the list of providers in the preview, maybe by passing them directly or in any other way?

Reply:
Agreed, I'll relook at all the previews to make them more demonstrative